src/components/ExhibitionRole/index.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { HTMLAttributes } from "react";
+import React, { ReactElement, HTMLAttributes } from "react";
 
 import RoleModel from "models/ExhibitionRole";
 
 type RoleProps = RoleModel & HTMLAttributes<HTMLElement>;
 
-const Role: React.FC<RoleProps> = ({ role }: RoleProps) => (
+const Role = ({ role, className }: RoleProps): ReactElement => (
```

Comment:
`className` attribute is not referenced in function. Should be removed @sophiewongsp